lab_2/main.py

```diff
@@ -24,7 +31,14 @@ def create_zero_matrix(rows: int, columns: int) -> list:
     e.g. rows = 2, columns = 2
     --> [[0, 0], [0, 0]]
     """
-    pass
+    if not isinstance(rows, int) or not isinstance(columns, int) or \
+            isinstance(rows, bool) or isinstance(columns, bool):
+        return []
+    zero_matrix = []
+    n_columns = [0] * columns
+    if n_columns:
```

Comment:
why do you need this check?

Reply:
I need it because if columns <= 0 I get [], not [0], so i use this check to avoid empty list in the matrix generation

Reply:
OK